maintenance-branch

```diff
@@ -1 +1 @@
-4-0-dev
\ No newline at end of file
+remove-monkey-patching
```

Comment:
Is there a need for parity? whats the Pr ordering here?

Reply:
Core first, then Mocks or Expectations no matter the order.

Reply:
Then as this is core, this should be dropped?

Reply:
Rolled it back.